doc/source/api/core/solver/fielddata.rst

```diff
@@ -29,104 +28,118 @@ The field data object is an attribute of the solver object:
 
 
-You can call the same method to get the corresponding surface face normals:
+You can call the same method to get the corresponding surface face-connectivity, normals and centroids
+by specifying ``FacesConnectivity``, ``FacesNormal`` and ``FacesCentroid`` respectively for the data_type.
 
 .. code-block:: python
 
-  >>> field_data.get_surface_data(surface_name="wall-elbow", data_type=SurfaceDataType.FacesNormal)
-  {0: array([ 1.3146671e-06, -1.1612752e-06,  5.7278072e-08, ...,
-       -2.6233408e-08,  1.6844007e-05,  1.0690871e-06], dtype=float32)}
-
+  >>> field_data.get_surface_data(surface_name="inlet", data_type=SurfaceDataType.FacesConnectivity)
+  {4: array([  4,   3,   2, ..., 379, 382, 388])}
 
+  >>> field_data.get_surface_data(surface_name="inlet", data_type=SurfaceDataType.FacesNormal)
+  {4: array([ 0.0000000e+00,  2.5835081e-06,  ...,
+          2.7459005e-06,  0.0000000e+00,  0.0000000e+00,  3.0340884e-06], dtype=float32)}
+  
+  >>> field_data.get_surface_data(surface_name="inlet", data_type=SurfaceDataType.FacesCentroid)
+  {4: array([-0.34724122,  0.        , -0.0442204 , -0.34724477, ...,
+         -0.04050309, -0.34645557,  0.        , -0.04421487, -0.34646705], dtype=float32)}
+       
+Response contains a dictionary of surface IDs to numpy array of the requested field. 
+
+get_scalar_field_data
+~~~~~~~~~~~~~~~~~~~~~
 You can call ``get_scalar_field_data`` to get scalar field data, such as temperature:
 
 .. code-block:: python
 
-  >>> field_data.get_scalar_field_data(surface_name="wall-elbow", field_name="temperature")
-  {0: array([0., 0., 0., ..., 0., 0., 0.], dtype=float32)}
+  >>> field_data.get_scalar_field_data(surface_name="inlet", field_name="temperature")
+  {4: array([922.6778 , 923.7954 , 923.791  , 922.67017, 922.88135,  ...,
+         924.94556, 924.9451 , 924.9536 , 923.87366, 922.8334 , 922.82434], dtype=float32)}
+         
+Response contains a dictionary of surface IDs to numpy array of the requested field.          
 
-
-You can call ``get_vector_field_data`` to get vector scalar field data, such as velocity:
+get_vector_field_data
+~~~~~~~~~~~~~~~~~~~~~
+You can call ``get_vector_field_data`` to get vector field data. 
 
 .. code-block:: python
 
-  >>> field_data.get_vector_field_data(surface_name="wall-elbow", vector_field="velocity")
-  {0: (array([0., 0., 0., ..., 0., 0., 0.], dtype=float32), 1.0)}
-
+  >>> field_data.get_vector_field_data(surface_name="inlet", vector_field="velocity")
+  {4: (array([ 5.81938386e-01, -1.01187916e+01,  3.14891455e-03,  ...,
+        -6.49216697e-02,  1.44923580e+00, -1.04387817e+01], dtype=float32), 0.00012235096)}
+  
+Response is a dictionary of surface IDs to a tuple containing a numpy array of ``vector field`` and ``vector-scale``. 
 
-The above example employs a separate method for requesting each type of field:
+get_pathlines_field_data
+~~~~~~~~~~~~~~~~~~~~~~~~  
+You can call ``get_pathlines_field_data`` to get pathlines data. 
```

Comment:
```suggestion
You can call the ``get_pathlines_field_data`` method to get pathlines field data. 
```

Reply:
You need to add the word "pathlines" to the Vale accept list. When Vale is implemented for the repo, it should be here: C:\AnsysDev\GitRepos\PyAnsys\pyfluent-ui\doc\styles\Vocab\ANSYS\accept.txt. However, I pulled from "main" and am not seeing it?!